control-agent/text-to-speech/speech-detection.py

```diff
@@ -81,7 +100,7 @@ def recognize_intent():
         return ""
```

Comment:
Consider putting in a more useful error message if this fails

Reply:
added